src/modules/solution/solution.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { JwtStrategy } from '../core/auth/jwt/jwt.strategy';
+import { PrismaService } from '../core/database/prisma/prisma.service';
+import { SolutionController } from './controllers/solution.controller';
+import { SolutionService } from './services/solution.service';
+
+@Module({
+  controllers: [SolutionController],
+  providers: [SolutionService, PrismaService, JwtStrategy]
+})
+export class QuestionModule { }
```

Comment:
```suggestion
export class SolutionModule{ }
```
SolutionModule이 아닌가요?

Reply:
왘ㅋㅋㅋㅋㅋ 감사합니다!